src/routes/authRoutes.ts

```diff
@@ -0,0 +1,17 @@
+import { Hono } from 'hono';
+import { authController } from '@/controllers/authController.ts';
+import { errorHandler } from '@/utils/custom-error';
+import { validateSign } from '@/middlewares/authValidation';
+import { jwt } from 'hono/jwt';
+
+export const authRoutes = new Hono().basePath('/auth');
+
+authRoutes.onError(errorHandler);
+authRoutes.use(
+  '/page',
+  jwt({
```

Comment:
jwt is a middleware, so if you need it only for one route you can use it like this
```
authRoutes.get('/page', jwt({
  secret: 'mySecretKey',
}), authController.getPayload);
```